test/setup_test.go

```diff
@@ -409,6 +409,23 @@ func testSetup() {
 }
 
 func applyAndWaitForApplications(commitID string) {
+	// TODO: remove this block after #2129 is released
+	By("grafting dbre namespaces")
+	if doUpgrade {
+		ExecSafeAt(boot0, "argocd", "app", "set", "team-management", "--sync-policy", "none")
+		nss := []string{"app-cybozu-com-mysql", "app-cybozu-com-mysql-replica", "app-cybozu-com-mysql-restore", "app-mysql-tenant-service"}
+		for _, ns := range nss {
+			_, _, err := ExecAt(boot0, "kubectl", "get", "ns", ns)
+			if err == nil {
+				ExecSafeAt(boot0, "kubectl", "label", "ns", ns, "app.kubernetes.io/instance-")
+				ExecSafeAt(boot0, "kubectl", "accurate", "sub", "graft", ns, "team-dbre")
+			}
+			if ns != "app-cybozu-com-mysql-restore" {
+				ExecSafeAt(boot0, "kubectl", "patch", "subnamespace", "-n", "team-dbre", ns, "--type", "merge", "-p", `'{"spec":{"labels":{"cybozu.com/alert-group":"dbre"}}}'`)
+			}
```

Comment:
I suspect that SubNamespaces will not be created after merging this PR.
`kubectl patch` returns an error if the target object does not exist.

```suggestion
			if err == nil {
				ExecSafeAt(boot0, "kubectl", "label", "ns", ns, "app.kubernetes.io/instance-")
				ExecSafeAt(boot0, "kubectl", "accurate", "sub", "graft", ns, "team-dbre")
				if ns != "app-cybozu-com-mysql-restore" {
					ExecSafeAt(boot0, "kubectl", "patch", "subnamespace", "-n", "team-dbre", ns, "--type", "merge", "-p", `'{"spec":{"labels":{"cybozu.com/alert-group":"dbre"}}}'`)
				}
			}
```